src/test/java/io/anserini/document/JsonDocumentArrayTest.java

```diff
@@ -0,0 +1,70 @@
+/**
+ * Anserini: An information retrieval toolkit built on Lucene
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package io.anserini.document;
+
+import java.io.BufferedWriter;
+import java.io.File;
+import java.io.FileOutputStream;
+import java.io.OutputStreamWriter;
+import java.io.Writer;
+import java.util.HashMap;
+
+import org.junit.After;
+import org.junit.Before;
+
+
+public class JsonDocumentArrayTest extends DocumentTest {
+  private String sampleFile = "sampleJsonArray.json";
+
+  @Before
+  public void setUP() throws Exception {
```

Comment:
`setUP` -> `setup`

same below.

Reply:
Yes.
But I think it should be setUp()... it is a override method from `LuceneTestCase`